src/Drupal/Commands/core/CliCommands.php

```diff
@@ -92,25 +81,34 @@ public function cli(array $options = ['version-history' => FALSE]) {
    * @return array
    */
   protected function getDrushCommands() {
-    $commands = drush_get_commands();
+    $application = \Drush::getApplication();
+    //annotation_adapter_add_legacy_commands_to_application($application);
```

Comment:
@weitzman using the legacy command adapter here produces a ton of warnings (looks like one foe each command) like:

```
 [warning] array_key_exists() expects parameter 2 to be array, string given annotationcommand_adapter.inc:503
 [warning] Illegal string offset 'default_value' annotationcommand_adapter.inc:504
 [warning] Illegal string offset 'default_value' annotationcommand_adapter.inc:506
 [warning] Illegal string offset 'description' annotationcommand_adapter.inc:506
 [warning] Illegal string offset 'default_value' annotationcommand_adapter.inc:506
```